itests/kit/deals.go

```diff
@@ -87,6 +87,17 @@ func NewDealHarness(t *testing.T, client *TestFullNode, main *TestMiner, market
 //
 // TODO: convert input parameters to struct, and add size as an input param.
 func (dh *DealHarness) MakeOnlineDeal(ctx context.Context, params MakeFullDealParams) (deal *cid.Cid, res *api.ImportRes, path string) {
+	deal, res, path = dh.StartRandomDeal(ctx, params)
+
+	// TODO: this sleep is only necessary because deals don't immediately get logged in the dealstore, we should fix this
```

Comment:
Is there an issue you can link?  If not we should file one to track this work so we can throw it in the backlog and eventually get it done.

Reply:
This todo is massively old, I'm ~50% sure that this was fixed now

Reply:
If it's deprecated consider investigating and removing it